cli.js

```diff
@@ -4,13 +4,18 @@ const logSymbols = require('log-symbols');
 const meow = require('meow');
 const isOnline = require('is-online');
 
-meow(`
+const cli = meow(`
 	Example
 	  $ is-online
 	  ${logSymbols.success} Online
 `);
 
-isOnline().then(online => {
+const options = {};
+if (cli.flags.timeout) {
+	options.timeout = cli.flags.timeout;
+}
```

Comment:
Do you want me to add support for the `version` option as well (will have to shim it as `--ip-version`)?